sysdig/resource_sysdig_secure_drift_policy_test.go

```diff
@@ -67,9 +67,6 @@ resource "sysdig_secure_drift_policy" "sample" {
     prohibited_binaries {
       items = ["/usr/bin/curl"]
     }
-	process_based_exceptions {
```

Comment:
Please do not modify previous tests, as they ensure that existing code that customers may be using is not going to break.

Add a new test case with the new sysdig_secure_drift_policy.sample with the new fields you added to the resource.